\\ In file included from toolchain_build_i386/gcc-3.4.6/libiberty/hashtab.c:49:
\\ build_i386/staging_dir/i386-linux-uclibc/sys-include/malloc.h:193: error: ISO C forbids data definition with no type or storage class
Index: uClibc/include/malloc.h
===================================================================
--- uClibc/include/malloc.h	(revision 20307)
+++ uClibc/include/malloc.h	(working copy)
@@ -190,8 +190,6 @@ extern int mallopt __MALLOC_P ((int __pa
  * NB: do not use stdio in __uc_malloc_failed handler! */
 extern void *__uc_malloc(size_t size);
 extern void (*__uc_malloc_failed)(size_t size);
-libc_hidden_proto(__uc_malloc);
-libc_hidden_proto(__uc_malloc_failed);
 
 #ifdef __cplusplus
 } /* end of extern "C" */
Index: uClibc/libc/stdlib/__uc_malloc.c
===================================================================
--- uClibc/libc/stdlib/__uc_malloc.c	(revision 20241)
+++ uClibc/libc/stdlib/__uc_malloc.c	(working copy)
@@ -22,6 +22,9 @@ Cambridge, MA 02139, USA.
 #include <unistd.h>
 #include <malloc.h>
 
+libc_hidden_proto(__uc_malloc);
+libc_hidden_proto(__uc_malloc_failed);
+
 void (*__uc_malloc_failed)(size_t size) = NULL;
 /* Seemingly superfluous assigment of NULL above prevents gas error
  * ("__uc_malloc_failed can't be equated to common symbol
